api/src/main/java/edu/wpi/first/shuffleboard/api/components/EditableLabel.java

```diff
@@ -48,7 +48,6 @@ public EditableLabel() {
       if (isEditing) {
         editField.requestFocus();
       }
-      label.setVisible(!isEditing);
```

Comment:
?

Reply:
The visible property is already bound so all this does is throw an exception